src/Tribe/Cache.php

```diff
@@ -68,7 +70,7 @@ public function set( $id, $value, $expiration = 0, $expiration_trigger = '' ) {
 			// Add so we know what group to use in the future.
 			$this->non_persistent_keys[] = $key;
 		} else {
-			$group = 'tribe-events';
+			$group = self::PERSISTENT_GROUP;
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 8e0781f0c9def131ab6b3b7c318d27d7</sub>